### PR TITLE
Feature/pilot1

### PR DESCRIPTION
**Pull request by ajloughr-leidos**
# PR Details
## Description

Committing the changes made for Pilot 1

## How Has This Been Tested?

Tested during pilot 1

## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [x ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING**](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/Contributing.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: README.md
Comment (davidgayman) [inline README.md:1]: finish writing this

File: .gitignore
Comment (davidgayman) [inline .gitignore:1]: add config/node_info.config. That will still alllow it to be in the repo but changes will be ignored.

File: scripts/carla_python_scripts/MAP.json
Comment (davidgayman) [inline scripts/carla_python_scripts/MAP.json:1]: move this file to the maps directory

File: scripts/carla_python_scripts/destroy_signals.py
Comment (davidgayman) [inline scripts/carla_python_scripts/destroy_signals.py:1]: why was this script deleted? if it doesn't destroy signals the file should be removed.

File: scripts/carla_python_scripts/destroy_vehicles.py
Comment (davidgayman) [inline scripts/carla_python_scripts/destroy_vehicles.py:1]: why was this script deleted? if it doesn't destroy vehicles  the file should be removed.

File: scripts/run_scripts/pilot1/tena-mw-components/start-execution-manager.sh
Comment (davidgayman) [inline scripts/run_scripts/pilot1/tena-mw-components/start-execution-manager.sh:1]: remove?

File: scripts/run_scripts/pilot1/tena-mw-components/start-tena-data-collector.sh
Comment (davidgayman) [inline scripts/run_scripts/pilot1/tena-mw-components/start-tena-data-collector.sh:1]: do we need this?

**Review by davidgayman — CHANGES_REQUESTED** [review]

File: .gitignore
Comment (ajloughr-leidos) [reply .gitignore:1]: added

File: scripts/carla_python_scripts/MAP.json
Comment (ajloughr-leidos) [reply scripts/carla_python_scripts/MAP.json:1]: removed

File: scripts/carla_python_scripts/destroy_signals.py
Comment (ajloughr-leidos) [reply scripts/carla_python_scripts/destroy_signals.py:1]: this was a change to all scripts to use an external script to find the carla egg

File: scripts/carla_python_scripts/destroy_vehicles.py
Comment (ajloughr-leidos) [reply scripts/carla_python_scripts/destroy_vehicles.py:1]: same as destroy signals

File: scripts/run_scripts/pilot1/tena-mw-components/start-execution-manager.sh
Comment (ajloughr-leidos) [reply scripts/run_scripts/pilot1/tena-mw-components/start-execution-manager.sh:1]: will keep and config during test-4

File: scripts/run_scripts/pilot1/tena-mw-components/start-tena-data-collector.sh
Comment (ajloughr-leidos) [reply scripts/run_scripts/pilot1/tena-mw-components/start-tena-data-collector.sh:1]: will keep and config during test-4

add data replay?

File: README.md
Comment (ajloughr-leidos) [reply README.md:1]: david updated

**Review by davidgayman — APPROVED** [review]

**Pull request merged**